### ISSUE TITLE
Multistage alignment issue when stages transition from `shrink_processing_chunk: true` to `false`

### ISSUE THREAD

**Issue by nkemnitz** [ZettaAI/zetta_utils#410]
#BBOX: {
	"@type": "BBox3D.from_coords"
	start_coord: [0, 0, 3452]
	end_coord: [384, 512, 3453]
	resolution: [1024, 1024, 30]
}

Poor alignment happens at 64nm, which is the first stage that does not use `shrink_processing_chunk`. Maybe because using the 128nm field as input causes a weird offset change?

Issue is avoided by keeping `shrink_processing_chunk=False` for all resolutions.

**Comment by supersergiy** [comment]
@nkemnitz do you have an example spec to reproduce this?

**Comment by nkemnitz** [comment]
No. It used to be https://github.com/ZettaAI/zetta_utils/blob/nkem/zfish-enc/specs/nico/inference/zfish/0_encoding_zfish/1_Zfish_prepare_aced.cue#L79-L80 with the above bounding box.

But subchunkable has significantly changed since then with `expand_bbox_processing`, `expand_bbox_backend` and `expand_bbox_resolution`.

**Comment by supersergiy** [comment]
@nkemnitz is this still an issue?

**Comment by nkemnitz** [comment]
Think we can close this. There have been too many changes since then, alignment does not use `shrink_processing_chunk` anymore, and ZFish was a non-square dataset(?) - maybe that's why I ran into trouble...
